README.md

```diff
@@ -1,11 +1,11 @@
 SELinux Policy Cookbook
 ======================
-This cookbbok can be used to manage SELinux policies and components (rather than just enable / disable enforcing).  
+This cookbbok can be used to manage SELinux policies and components (rather than just enable / disable enforcing).
```

Comment:
Why you no EOL space? While this looks legit in GH markdown, it will break actual markdown like Supermarket's